syntax = "proto2";

message TopicConfig {
    optional string camera_M3JUVC_topic = 1;
    optional string camera_XYZ_topic = 2;
    optional string camera_merged_topic = 3;
}

message Config {
    optional TopicConfig topic_config = 1;
}
